SonarQube.Client/Api/V7_20/GetIssuesRequest.cs

```diff
@@ -79,7 +80,7 @@ protected override SonarQubeIssue[] ParseResponse(string response)
         }
 
         private SonarQubeIssue ToSonarQubeIssue(ServerIssue issue) =>
-            new SonarQubeIssue(ComputePath(issue.Component), issue.Hash, issue.Line, issue.Message, ComputeModuleKey(issue),
+            new SonarQubeIssue(FilePathNormalizer.NormalizeSonarQubePath(ComputePath(issue.Component)), issue.Hash, issue.Line, issue.Message, ComputeModuleKey(issue),
```

Comment:
I might have put the call to 'Normalize...' inside 'ComputePath' since it's part of the path processing, but it doesn't really make any difference.